vault/logical_system.go

```diff
@@ -1481,6 +1498,18 @@ func (b *SystemBackend) handleMount(ctx context.Context, req *logical.Request, d
 	pluginName := data.Get("plugin_name").(string)
 	sealWrap := data.Get("seal_wrap").(bool)
 
+	options := data.Get("options").(map[string]interface{})
+
+	optionMap := make(map[string]string)
```

Comment:
I'm seeing this in a couple of places, might be worth breaking this out into a `parseOptionsMap` func.